app/core/error_handlers.py

```
@@ -0,0 +1,36 @@
+        status_code=422,
+        content=format_error_response(422, str(exc.errors()), "validation_error"),
+    )
+
+async def db_exception_handler(request: Request, exc: sqlalchemy.exc.SQLAlchemyError):
+    logger.error(f"Database error: {str(exc)} | Path: {request.url}")
+    return JSONResponse(
+        status_code=500,
+        content=format_error_response(500, "A database error occurred.", "database_error"),
+    )
+
+async def generic_exception_handler(request: Request, exc: Exception):
```

Ruff (0.8.2)

Missing return type annotation for public function `generic_exception_handler`

(ANN201)